File: 2/arithmetic.py
#!python

mealCost = float(input())  # original meal price
tipPercent = int(input())  # tip percentage
taxPercent = int(input())  # tax percentage

# Write your calculation code here.
cost = mealCost * (1 + (tipPercent + taxPercent) / 100.0)

# cast the result of the rounding operation to an int and save it as totalCost
totalCost = round(cost)

# Print your result
print("The total meal cost is " + str(totalCost) + " dollars.")

File: 2/Arithmetic.java
package arithmetic;

import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;
public class Arithmetic {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double mealCost = scan.nextDouble(); // original meal price
        int tipPercent = scan.nextInt(); // tip percentage
        int taxPercent = scan.nextInt(); // tax percentage
        scan.close();
      
        // Write your calculation code here.
        double cost = mealCost * (1 + (tipPercent + taxPercent) / 100.0);
        // cast the result of the rounding operation to an int and save it as totalCost 
        int totalCost = (int) Math.round(cost);
      
        // Print your result
        System.out.println("The total meal cost is " + totalCost + " dollars.");
    }
}
